src/main/java/tutorlink/assignmentpackage/Assignment.java

```diff
@@ -49,4 +49,50 @@ public String toString() {
     public double getWeighting() {
         return weighting;
     }
+
```

Comment:
Thanks for helping to refactor the code so far. 

I understand that we are trying to consolidate our code first, but is there a reason why you chose to move the contents of the enum file into the Assignment.java file?

Reply:
I'm assuming we will use this later, and since the grade is for the entire course, it would make sense moving it either Assignment or AssignmentList as is it most closely related to them. I choose Assignment

Reply:
LGTM, we can always modify it further down the line if required.